js/templates/pageNav.html

```diff
@@ -43,16 +43,45 @@
           <% } %>
           ></a>
           <nav class="navList listBox clrBr clrP shad1 foldDown js-navList js-navPopMenu">
-            <a class="listLink">Current Node <i class="ion-android-arrow-dropdown floR"></i> </a>
-            <hr class="unleaded">
-            <a class="listLink"><i class="ion-android-bulb"></i> Foo <span class="floR clrT2">Cltrl + ?</span> </a>
-            <a class="listLink"><i class="ion-android-cloud-circle"></i> Foo</a>
-            <hr>
-            <a href="#transactions" class="listLink"><i class="ion-log-out"></i> Purchases</a>
-            <a href="#transactions" class="listLink"><i class="ion-log-in"></i> Sales</a>
-            <a href="#transactions" class="listLink"><i class="ion-briefcase"></i> Cases</a>
-            <hr>
-            <a class="listLink js-navSettings"><i class="ion-toggle-filled"></i> Settings</a>
+            <div class="listGroup clrS clrBr">
+              <a class="listLink clrS clrBr">Current Node <i
+                class="ion-arrow-right-b floR"></i> </a>
+            </div>
+            <div class="listGroup clrS clrBr">
+              <a class="listLink">
+                <span>My Page</span><span class="clrT2">Cltrl + ?</span>
```

Comment:
Really nice work on supporting text that breaks to multiple lines:

![image](https://cloud.githubusercontent.com/assets/794517/17120232/cbb68160-527f-11e6-85fb-131df10240d5.png)

One minor thing that I won't hold up the merge for is if we come across a really long single word. But, it's entirely possible we won't even come upon this case, so we could totally cross that bridge if we come to it.

![image](https://cloud.githubusercontent.com/assets/794517/17120267/01743c7a-5280-11e6-9a73-932114b286f1.png)